CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # HEAD
 
+# [v2.5.0](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/releases/tag/v2.5.0)
+
+Image updates:
+
+* change log level to V5 when dicovering path not match pattern by @hellogdc in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/235
+* Update Klog to V2 by @Kartik494 in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/234
+* Duplicate volume guard by @davidmccormick in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/246
+* add local volume discovery period flag by @dabaooline in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/261
+* Multi linux arch and multi windows distro builds by @mauriciopoppe in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/273
+* Update discovery and deletion code to work in Windows nodes through CSI Proxy by @mauriciopoppe in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/276
+* feat: add lstc2022 windows image build by @andyzhangx in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/314
+* fix: golang.org/x/crypto for CVE-2022-27191 by @andyzhangx in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/321
+* feat: support namePattern as a list by @andyzhangx in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/320
+
+Deployment updates:
```

Comment:
@mauriciopoppe not sure where you want to put helm chart release notes now that it's separate.

Reply:
I'll create a PR to add a CHANGELOG.md in the helm chart root.